### PR TITLE
FINERACT-1724: Add disallowExpectedDisbursements flag in Loan account response

### PR DESCRIPTION
**Pull request by josehernandezfintecheandomx**
## Description

Add `disallowExpectedDisbursements` flag in Loan account API response

[FINERACT-1724](https://issues.apache.org/jira/browse/FINERACT-1724).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java [hunk 1]
```diff
@@ -127,6 +126,7 @@ public final class LoanAccountData {
     private final LocalDate interestChargedFromDate;
     private final LocalDate expectedFirstRepaymentOnDate;
     private final Boolean syncDisbursementWithMeeting;
+    private final Boolean disallowExpectedDisbursements;
```
Comment (galovics) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java:129]: Please add this attribute to the Avro schema as well.

**Review by galovics — CHANGES_REQUESTED** [review]

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java [hunk 2]
```diff
@@ -1307,10 +1299,6 @@ public static LoanAccountData associationsAndTemplate(final LoanAccountData acc,
             final PaidInAdvanceData paidInAdvance, Collection<InterestRatePeriodData> interestRatesPeriods,
             final Collection<LoanAccountSummaryData> clientActiveLoanOptions, final List<RateData> rates, final Boolean isRatesEnabled,
             final CollectionData delinquent) {
-        LoanProductConfigurableAttributes loanProductConfigurableAttributes = null;
```
Comment (adamsaghy) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java:1310]: is there a reason to remove these?

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java [hunk 3]
```diff
@@ -1080,9 +1074,6 @@ public static LoanAccountData loanProductWithTemplateDefaults(final LoanProductD
         final PaidInAdvanceData paidInAdvance = null;
         final Collection<InterestRatePeriodData> interestRatesPeriods = null;
 
-        final Boolean isVariableInstallmentsAllowed = Boolean.FALSE;
```
Comment (adamsaghy) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java:1083]: is there a reason to remove these?

**Comment by adamsaghy** [comment]
@josehernandezfintecheandomx Please check the failing tests!

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java (hunk 2 above)
Comment (josehernandezfintecheandomx) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java:1310]: Without use

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java (hunk 3 above)
Comment (josehernandezfintecheandomx) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java:1083]: Without use

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java (hunk 1 above)
Comment (josehernandezfintecheandomx) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java:129]: Can someone explain me that?

Comment (josehernandezfintecheandomx) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java:129]: Done!

**Review by galovics — APPROVED** [review]

**Pull request merged**